packages/transaction-metrics-exporter/package.json

```diff
@@ -8,7 +8,7 @@
   "private": true,
   "dependencies": {
     "@celo/contractkit": "0.2.4",
-    "@celo/utils": "^0.1.0",
+    "@celo/utils": "0.1.5-dev",
```

Comment:
This package is built as a docker container... not sure if that docker has a copy of git's version of utils. If not it should use a published version. Probably the best anyway

Reply:
It builds from source, so this is safe https://github.com/celo-org/celo-monorepo/blob/cbdd336341a5d63bd06d232e0056f71320b34252/dockerfiles/transaction-metrics-exporter/Dockerfile#L27